packages/tab/index.tsx

```diff
@@ -59,6 +60,7 @@ export default class Tab extends React.Component<TabProps, TabState> {
 
   static defaultProps: Partial<TabProps> = {
     active: false,
+    focusOnActivate: false,
```

Comment:
I can change this to `true`, but I'm wondering what the reasoning behind this default is since it seems to me that focusing on activate is the non-standard scenario.

Tabs that are activated via mouse or keyboard navigation will already have focus so this setting mainly affects activation that happens a different way, e.g. on component mount. I think for accessibility reasons tabs should not be stealing focus by default but rather it should be something the developer explicitly opts into.

Reply:
Ya I definitely agree with you, and it cost me a few hours tracking down what was going on with our catalog app. However if we don't change this to true, we are initially out of sync with MDC Web's foundational layer, which causes bugs.

I think we should open an issue with MDC Web and then reflect it back here. What do you think?

Reply:
Sure, that seems reasonable.

Reply:
@ben-mckernan I actually talked to the team. The answer for default true is a11y. If a tab is programatically activated, the screenreader should know. Without focusing, you'd need to implement aria-live, which is not the recommended way of handling a11y things.

Reply:
Ah ok, interesting. I definitely need to read up more on these a11y recommendations it seems. Thanks for letting me know!